
2025-02-01 12:20:07,178 - root - ERROR - Error while checking user credentials: 172.10.10.99:27017: [Errno 61] Connection refused (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 679dc400ce1a38cffa18753d, topology_type: Unknown, servers: [<ServerDescription ('172.10.10.99', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('172.10.10.99:27017: [Errno 61] Connection refused (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
2025-02-01 12:20:47,528 - root - INFO - User mani4 authenticated successfully.
2025-02-01 12:21:01,377 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:21:01,417 - ApplicationManagment.evaluation - INFO - Generated process ID: 004006be
2025-02-01 12:21:01,417 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 004006be
2025-02-01 12:21:01,440 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: Invalid payload format provided
2025-02-01 12:21:01,448 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 12:21:01,470 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 12:26:29,816 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:26:29,853 - ApplicationManagment.evaluation - INFO - Generated process ID: 7e53328f
2025-02-01 12:26:29,854 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 7e53328f
2025-02-01 12:28:11,640 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-8' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:53> exception=HTTPException(status_code=500, detail='Internal server error: ')>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 613, in get_results
    raise HTTPException(status_code=404, detail="Document not found.")
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 161, in background_evaluation
    all_results = await self.mongoHandler.get_results(process_id)
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 626, in get_results
    raise HTTPException(status_code=500, detail=str(e))
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 182, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 12:28:48,748 - root - INFO - User mani4 authenticated successfully.
2025-02-01 12:29:00,141 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:29:10,814 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:29:10,871 - ApplicationManagment.evaluation - INFO - Generated process ID: feb51796
2025-02-01 12:29:10,872 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: feb51796
2025-02-01 12:29:55,532 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 12:42:16,503 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-13' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:53> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 166, in background_evaluation
    resultpath = excelConverter.convert_json_to_excel(all_results, EvaluationHandler.results_path, self.config_type)
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 182, in background_evaluation
    await self.mongoHandler.update_status_record(status_record)
fastapi.exceptions.HTTPException
2025-02-01 12:42:33,795 - root - INFO - User mani4 authenticated successfully.
2025-02-01 12:42:39,850 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:42:39,893 - ApplicationManagment.evaluation - INFO - Generated process ID: fd279a9d
2025-02-01 12:42:39,894 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: fd279a9d
2025-02-01 12:44:21,859 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: 500 Server Error: Internal Server Error for url: http://172.10.10.26:4001/job/2878b151-2930-4c6f-8114-53c294fbb47a
2025-02-01 12:44:21,860 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: 500 Server Error: Internal Server Error for url: http://172.10.10.26:4001/job/458164b7-68a4-4ba7-b9e4-828bb85565bf
2025-02-01 12:44:21,861 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: 500 Server Error: Internal Server Error for url: http://172.10.10.26:4001/job/74933a2e-a9de-4858-ac1d-d9fe8de70e2d
2025-02-01 12:44:21,861 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: 
2025-02-01 12:44:21,861 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 12:44:21,861 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 12:44:21,872 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 12:44:21,895 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 12:46:07,249 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-10' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail='Internal server error: ')>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 613, in get_results
    raise HTTPException(status_code=404, detail="Document not found.")
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 162, in background_evaluation
    all_results = await self.mongoHandler.get_results(process_id)
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 626, in get_results
    raise HTTPException(status_code=500, detail=str(e))
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 12:46:26,984 - root - INFO - User mani4 authenticated successfully.
2025-02-01 12:46:35,515 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:46:35,554 - ApplicationManagment.evaluation - INFO - Generated process ID: a7d0538f
2025-02-01 12:46:35,554 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: a7d0538f
2025-02-01 12:46:46,891 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: 500 Server Error: Internal Server Error for url: http://172.10.10.26:4001/job/57d29080-55e1-4684-838f-0f6a7dbcc7a9
2025-02-01 12:46:46,892 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: 500 Server Error: Internal Server Error for url: http://172.10.10.26:4001/job/e4f059fe-e7c8-4d01-a66e-0f6d2e9fdd9f
2025-02-01 12:46:46,892 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: 500 Server Error: Internal Server Error for url: http://172.10.10.26:4001/job/3a4e8814-1c8e-4379-a804-08c059f1c560
2025-02-01 12:46:46,892 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: 
2025-02-01 12:46:46,892 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 12:46:46,893 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 12:46:46,901 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 12:46:46,923 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 12:49:54,294 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:49:54,337 - ApplicationManagment.evaluation - INFO - Generated process ID: e2aef3a9
2025-02-01 12:49:54,338 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: e2aef3a9
2025-02-01 12:50:38,708 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 12:53:45,936 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-10' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail='Internal server error: ')>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 613, in get_results
    raise HTTPException(status_code=404, detail="Document not found.")
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 162, in background_evaluation
    all_results = await self.mongoHandler.get_results(process_id)
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 626, in get_results
    raise HTTPException(status_code=500, detail=str(e))
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 12:53:45,937 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-13' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 12:54:03,179 - root - INFO - User mani4 authenticated successfully.
2025-02-01 12:54:07,396 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:54:07,441 - ApplicationManagment.evaluation - INFO - Generated process ID: d33ec16a
2025-02-01 12:54:07,442 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: d33ec16a
2025-02-01 12:56:21,509 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 12:58:31,310 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-8' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 12:58:45,890 - root - INFO - User mani4 authenticated successfully.
2025-02-01 12:58:51,018 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 12:58:51,060 - ApplicationManagment.evaluation - INFO - Generated process ID: 8229f854
2025-02-01 12:58:51,061 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 8229f854
2025-02-01 12:59:34,560 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 13:00:30,191 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-8' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 13:00:48,071 - root - INFO - User mani4 authenticated successfully.
2025-02-01 13:00:52,888 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 13:00:52,931 - ApplicationManagment.evaluation - INFO - Generated process ID: 5869597b
2025-02-01 13:00:52,931 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 5869597b
2025-02-01 13:01:36,437 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 13:05:47,606 - root - INFO - User mani4 authenticated successfully.
2025-02-01 13:06:57,059 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 13:06:57,105 - ApplicationManagment.evaluation - INFO - Generated process ID: 6d5a40c2
2025-02-01 13:06:57,105 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 6d5a40c2
2025-02-01 13:06:57,128 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: Invalid payload format provided
2025-02-01 13:06:57,136 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 13:06:57,156 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 13:09:19,514 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 13:09:19,559 - ApplicationManagment.evaluation - INFO - Generated process ID: 6e26e8f5
2025-02-01 13:09:19,559 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 6e26e8f5
2025-02-01 13:11:32,693 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 13:30:41,241 - root - INFO - User mani4 authenticated successfully.
2025-02-01 13:30:45,812 - root - INFO - Accessing collection 'LLMPrompts' in database: applicationDB
2025-02-01 13:30:45,813 - root - INFO - json_data to be inserted: {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}
2025-02-01 13:30:45,837 - root - INFO - Inserting data into 'LLMPrompts' collection.
2025-02-01 13:30:45,844 - root - INFO - Prompt added successfully with ID: 679dd4ad53d30eb1cd6a673f
2025-02-01 13:30:45,844 - AiManagement.prompts - INFO - Prompt added successfully for clientApiKey sfdfsf
2025-02-01 13:30:51,629 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-01 13:30:51,630 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-01 13:31:26,742 - root - INFO - User mani4 authenticated successfully.
2025-02-01 13:32:14,059 - root - ERROR - Error loading YAML data: expected str, bytes or os.PathLike object, not dict
2025-02-01 13:32:14,059 - asyncio - ERROR - Unexpected error in metrics calculation: 
2025-02-01 13:34:23,434 - root - ERROR - Error loading YAML data: [Errno 2] No such file or directory: 'C:/Users/Admin/projects/Model_Evaluation/services/Evaluation/payload/result.yml'
2025-02-01 13:34:23,435 - asyncio - ERROR - Unexpected error in metrics calculation: 
2025-02-01 13:39:57,606 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-9' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 13:39:57,608 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-14' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail='Internal server error: ')>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 613, in get_results
    raise HTTPException(status_code=404, detail="Document not found.")
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 162, in background_evaluation
    all_results = await self.mongoHandler.get_results(process_id)
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/Database/evaluationSetup.py", line 626, in get_results
    raise HTTPException(status_code=500, detail=str(e))
fastapi.exceptions.HTTPException

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 13:39:57,608 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-19' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 13:41:09,641 - root - INFO - User mani4 authenticated successfully.
2025-02-01 13:41:33,164 - asyncio - ERROR - Unexpected error in metrics calculation: 'results_path'
2025-02-01 13:43:28,823 - root - INFO - User mani4 authenticated successfully.
2025-02-01 13:48:24,811 - huggingface_hub.file_download - WARNING - Error while downloading from https://cdn-lfs.hf.co/roberta-large/047c85f0b96269cd62e6f732644f067004eebd95af5b5d35965ae2528f13bf38?response-content-disposition=inline%3B+filename*%3DUTF-8%27%27model.safetensors%3B+filename%3D%22model.safetensors%22%3B&Expires=1738401259&Policy=eyJTdGF0ZW1lbnQiOlt7IkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MTczODQwMTI1OX19LCJSZXNvdXJjZSI6Imh0dHBzOi8vY2RuLWxmcy5oZi5jby9yb2JlcnRhLWxhcmdlLzA0N2M4NWYwYjk2MjY5Y2Q2MmU2ZjczMjY0NGYwNjcwMDRlZWJkOTVhZjViNWQzNTk2NWFlMjUyOGYxM2JmMzg%7EcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj0qIn1dfQ__&Signature=NOjO9CPLA0Cm2GnNr8MoAxOA2K5kkmJNY1RPPvDaNyhlHebuOTuErLV3pfF-oifPjeNYZrhvybSqE7kkdOxSFVl6xwBetyzQm3uYn2lbeQyjtEN%7EVlluBLSu-ZzMemPH276%7Ev9vgdFd8TFKyeT2EF8uxHMXaNoaJjqgS31VilPHTbKi9gEOe3T5ghwEUqrio5Eh8TTWgX%7EqSl1A8DoPJ-Bjl5ar8mv3BtwiEHlA0lgkszo9ndU8vVQD0iIUOsnFCsrPyQxPUYuOI0Wv6VjusA70fZZpWBb7NnnG3AsbuZzj%7EO-EELEVEJxzCxodUlckjvnMLnYTcbiAMSYQA853JKQ__&Key-Pair-Id=K3RPWS32NSSJCE: HTTPSConnectionPool(host='cdn-lfs.hf.co', port=443): Read timed out.
Trying to resume download...
2025-02-01 13:48:49,256 - absl - INFO - Using default tokenizer.
2025-02-01 13:48:49,271 - asyncio - INFO - Updating model status to Completed...
2025-02-01 14:03:36,418 - root - INFO - User mani4 authenticated successfully.
2025-02-01 15:08:43,728 - root - INFO - User mani4 authenticated successfully.
2025-02-01 15:09:08,657 - root - INFO - User mani4 authenticated successfully.
2025-02-01 15:56:42,994 - root - INFO - User mani4 authenticated successfully.
2025-02-01 15:57:33,329 - root - INFO - User mani4 authenticated successfully.
2025-02-01 16:22:09,335 - root - INFO - User mani4 authenticated successfully.
2025-02-01 16:22:25,905 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 16:22:25,955 - ApplicationManagment.evaluation - INFO - Generated process ID: d8350b3a
2025-02-01 16:22:25,955 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: d8350b3a
2025-02-01 16:24:53,979 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 16:31:34,694 - root - INFO - User superadmin authenticated successfully.
2025-02-01 16:31:37,047 - root - INFO - No org found in application database.
2025-02-01 16:37:45,022 - root - INFO - User admin authenticated successfully.
2025-02-01 16:37:47,237 - root - ERROR - Error while retrieving spaces: 'NoneType' object is not subscriptable
2025-02-01 16:38:31,091 - root - ERROR - Error while retrieving spaces: 'NoneType' object is not subscriptable
2025-02-01 16:38:36,885 - root - ERROR - Error while retrieving spaces: 'NoneType' object is not subscriptable
2025-02-01 16:38:53,915 - root - INFO - User superadmin authenticated successfully.
2025-02-01 16:38:54,641 - root - INFO - No org found in application database.
2025-02-01 16:39:17,752 - root - INFO - Organization sales created successfully with org id dfmx
2025-02-01 16:39:17,857 - root - INFO - Creating new database 'dfmx'.
2025-02-01 16:39:17,864 - root - INFO - No spaces found for this Org.
2025-02-01 16:39:52,200 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-83' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 16:40:02,433 - root - INFO - User admin authenticated successfully.
2025-02-01 16:40:03,607 - root - ERROR - Error while retrieving spaces: 'NoneType' object is not subscriptable
2025-02-01 16:40:15,647 - root - ERROR - Error while retrieving spaces: 'NoneType' object is not subscriptable
2025-02-01 16:40:40,726 - root - INFO - User mani4 authenticated successfully.
2025-02-01 16:41:00,012 - root - ERROR - Error while retrieving spaces: 'NoneType' object is not subscriptable
2025-02-01 16:41:16,452 - root - INFO - User superadmin authenticated successfully.
2025-02-01 16:41:17,715 - root - INFO - Creating new database 'dfmx'.
2025-02-01 16:41:17,721 - root - INFO - No spaces found for this Org.
2025-02-01 16:42:06,615 - root - INFO - User mani4 authenticated successfully.
2025-02-01 16:45:54,980 - root - INFO - User superadmin authenticated successfully.
2025-02-01 16:45:55,538 - root - INFO - Creating new database 'dfmx'.
2025-02-01 16:45:55,549 - root - INFO - No spaces found for this Org.
2025-02-01 16:46:02,773 - root - INFO - Creating new database 'dfmx'.
2025-02-01 16:46:02,779 - root - INFO - No spaces found for this Org.
2025-02-01 16:46:03,645 - root - INFO - No org found in application database.
2025-02-01 16:46:06,431 - root - INFO - No org found in application database.
2025-02-01 16:46:18,863 - root - INFO - User superadmin authenticated successfully.
2025-02-01 16:46:19,394 - root - INFO - No org found in application database.
2025-02-01 17:01:31,405 - root - INFO - User mani4 authenticated successfully.
2025-02-01 17:03:07,642 - root - INFO - Invalid Credentials: Incorrect password.
2025-02-01 17:03:11,839 - root - INFO - Invalid Credentials: Incorrect password.
2025-02-01 17:03:44,917 - root - INFO - User mani4 authenticated successfully.
2025-02-01 17:26:13,042 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 17:26:13,076 - ApplicationManagment.evaluation - INFO - Generated process ID: 17162507
2025-02-01 17:26:13,077 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 17162507
2025-02-01 17:28:04,920 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 17:28:04,962 - ApplicationManagment.evaluation - INFO - Generated process ID: ae5d6eb2
2025-02-01 17:28:04,962 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: ae5d6eb2
2025-02-01 17:28:48,417 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 17:29:10,451 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 17:50:03,245 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 17:50:03,291 - ApplicationManagment.evaluation - INFO - Generated process ID: a0f4a902
2025-02-01 17:50:03,291 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: a0f4a902
2025-02-01 17:52:17,993 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 'path'
2025-02-01 18:03:03,436 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-188' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 18:03:03,438 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-194' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 18:03:03,438 - asyncio - ERROR - Task exception was never retrieved
future: <Task finished name='Task-254' coro=<EvaluationHandler.background_evaluation() done, defined at /Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py:54> exception=HTTPException(status_code=500, detail="Internal server error: 'path'")>
Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 167, in background_evaluation
    await self.mongoHandler.update_results_path(process_id, resultpath["path"])
KeyError: 'path'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/apple/Documents/Genx-Platform-Backend/AIPlatform_backend/ApplicationManagment/Handlers/evaluationHandler.py", line 183, in background_evaluation
    raise HTTPException(status_code=500, detail=f"Internal server error: {e}")
fastapi.exceptions.HTTPException
2025-02-01 18:48:00,118 - root - INFO - User mani4 authenticated successfully.
2025-02-01 18:51:02,080 - root - INFO - User mani4 authenticated successfully.
2025-02-01 18:51:28,193 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 18:51:28,235 - ApplicationManagment.evaluation - INFO - Generated process ID: f4009aff
2025-02-01 18:51:28,235 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: f4009aff
2025-02-01 18:51:34,466 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: Transcription failed: HTTPConnectionPool(host='172.10.10.26', port=6003): Max retries exceeded with url: /llm/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f38f2ca2df0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-02-01 18:51:34,466 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 18:51:34,466 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 18:51:34,479 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 18:51:34,500 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 19:00:12,766 - root - INFO - User mani4 authenticated successfully.
2025-02-01 19:00:27,534 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 19:00:27,575 - ApplicationManagment.evaluation - INFO - Generated process ID: ccb28616
2025-02-01 19:00:27,575 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: ccb28616
2025-02-01 19:00:33,727 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: Transcription failed: HTTPConnectionPool(host='172.10.10.26', port=6003): Max retries exceeded with url: /llm/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f38f23e2ac0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-02-01 19:00:33,727 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 19:00:33,727 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 19:00:33,737 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 19:00:33,757 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 19:01:11,575 - absl - INFO - Using default tokenizer.
2025-02-01 19:01:11,590 - asyncio - INFO - Updating model status to Completed...
2025-02-01 19:22:13,070 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 19:22:13,115 - ApplicationManagment.evaluation - INFO - Generated process ID: dd60e985
2025-02-01 19:22:13,115 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: dd60e985
2025-02-01 19:22:19,565 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: Transcription failed: HTTPConnectionPool(host='172.10.10.26', port=6003): Max retries exceeded with url: /llm/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fca42154f40>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-02-01 19:22:19,565 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 19:22:19,565 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 19:22:19,584 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 19:22:19,632 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 19:25:10,731 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-01 19:25:10,777 - ApplicationManagment.evaluation - INFO - Generated process ID: cdc12713
2025-02-01 19:25:10,778 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: cdc12713
2025-02-01 19:25:39,057 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: Transcription failed: HTTPConnectionPool(host='172.10.10.26', port=6003): Max retries exceeded with url: /llm/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fca420ca460>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-02-01 19:25:39,057 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 19:25:39,058 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-01 19:25:39,068 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679cc7b2ee77a1870ffc93f0: Evaluation failed for model
2025-02-01 19:25:39,091 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-01 19:26:18,387 - absl - INFO - Using default tokenizer.
2025-02-01 19:26:18,393 - asyncio - INFO - Updating model status to Completed...
2025-02-01 19:29:03,186 - absl - INFO - Using default tokenizer.
2025-02-01 19:29:03,192 - asyncio - INFO - Updating model status to Completed...
2025-02-03 10:51:06,392 - root - INFO - Invalid Credentials: Incorrect password.
2025-02-03 10:51:09,388 - root - INFO - Invalid Credentials: Incorrect password.
2025-02-03 10:52:08,019 - root - INFO - User mani4 authenticated successfully.
2025-02-03 12:16:08,418 - root - INFO - User mani4 authenticated successfully.
2025-02-03 12:16:09,778 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 12:16:09,779 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 12:16:09,820 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 12:16:09,821 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 13:18:02,761 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-03 13:18:02,839 - ApplicationManagment.evaluation - INFO - Generated process ID: 85847697
2025-02-03 13:18:02,842 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 85847697
2025-02-03 13:19:17,911 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x158fd5f70>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 13:19:17,912 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x158fd5a60>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 13:19:17,912 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x158fd5250>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 13:19:17,913 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: 
2025-02-03 13:19:17,913 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-03 13:19:17,913 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-03 13:19:17,926 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679e4ef847b54d70c2e918ae: Evaluation failed for model
2025-02-03 13:19:17,964 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-03 14:07:16,300 - root - INFO - User mani4 authenticated successfully.
2025-02-03 14:41:35,624 - root - INFO - User mani authenticated successfully.
2025-02-03 14:41:38,554 - root - ERROR - Error while retrieving spaces: 'orgIds'
2025-02-03 14:42:14,611 - root - INFO - User mani4 authenticated successfully.
2025-02-03 14:42:16,964 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:42:16,965 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:42:17,099 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:42:17,100 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:42:50,578 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:42:50,580 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:42:50,654 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:42:50,654 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:43:11,886 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:43:11,886 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:43:11,961 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:43:11,961 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:45:09,183 - root - INFO - User mani4 authenticated successfully.
2025-02-03 14:45:09,559 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:45:09,559 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:45:09,629 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:45:09,629 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:47:18,051 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:47:18,054 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:47:18,143 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:47:18,143 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:47:44,231 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:47:44,232 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:47:44,290 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:47:44,290 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:48:14,466 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:48:14,468 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:48:14,540 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:48:14,540 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:49:11,507 - root - INFO - User mani4 authenticated successfully.
2025-02-03 14:49:11,916 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:49:11,916 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:49:11,983 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:49:11,983 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:49:23,148 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:49:23,149 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:49:23,219 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:49:23,220 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:49:34,349 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:49:34,351 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:49:34,407 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:49:34,408 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:50:42,343 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:50:42,343 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:50:42,416 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:50:42,416 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:51:56,727 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:51:56,728 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:51:56,804 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:51:56,804 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:52:55,183 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-03 14:52:55,226 - ApplicationManagment.evaluation - INFO - Generated process ID: 72450d6f
2025-02-03 14:52:55,227 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 72450d6f
2025-02-03 14:53:23,905 - root - INFO - User mani4 authenticated successfully.
2025-02-03 14:53:24,301 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:53:24,302 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:53:24,376 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 14:53:24,377 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 14:54:10,301 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x15a50aee0>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 14:54:10,302 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x15a50aac0>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 14:54:10,302 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x15a50a700>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 14:54:10,304 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: 
2025-02-03 14:54:10,305 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-03 14:54:10,305 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-03 14:54:10,314 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679e4ef847b54d70c2e918ae: Evaluation failed for model
2025-02-03 14:54:10,344 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-03 14:54:37,970 - ApplicationManagment.evaluation - INFO - MongoDBHandler initialized for orgId: buup
2025-02-03 14:54:38,030 - ApplicationManagment.evaluation - INFO - Generated process ID: 6fb63b27
2025-02-03 14:54:38,030 - ApplicationManagment.evaluation - INFO - Evaluation task started with process ID: 6fb63b27
2025-02-03 14:55:53,082 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x15a50a580>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 14:55:53,083 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x15a446c10>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 14:55:53,083 - ApplicationManagment.Handlers.evaluationHandler - ERROR - RequestException: HTTPConnectionPool(host='172.10.10.26', port=4001): Max retries exceeded with url: /accelerator/server (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x15a446fd0>: Failed to establish a new connection: [Errno 60] Operation timed out'))
2025-02-03 14:55:53,084 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Request failed: 
2025-02-03 14:55:53,084 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-03 14:55:53,084 - ApplicationManagment.Handlers.evaluationHandler - ERROR - An error occurred: 
2025-02-03 14:55:53,096 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Error during evaluation of model 679e4ef847b54d70c2e918ae: Evaluation failed for model
2025-02-03 14:55:53,124 - ApplicationManagment.Handlers.evaluationHandler - ERROR - Unexpected error in evaluation: 
2025-02-03 14:57:46,566 - absl - INFO - Using default tokenizer.
2025-02-03 14:57:46,568 - asyncio - INFO - Updating model status to Completed...
2025-02-03 15:01:07,843 - absl - INFO - Using default tokenizer.
2025-02-03 15:01:07,857 - asyncio - INFO - Updating model status to Completed...
2025-02-03 15:02:49,367 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:02:49,369 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:02:49,433 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:02:49,434 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:31,223 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:31,226 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:31,300 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:31,301 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:45,365 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:45,366 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:45,426 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:45,427 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:50,984 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:50,985 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:51,045 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:51,046 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:51,164 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:51,165 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:55,125 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:55,125 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:55,199 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:55,200 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 15:08:55,342 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 15:08:55,342 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 16:01:23,190 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 16:01:23,193 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 16:01:23,269 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 16:01:23,269 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 16:23:45,128 - root - INFO - User mani4 authenticated successfully.
2025-02-03 16:23:46,471 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 16:23:46,472 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 16:23:46,545 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 16:23:46,545 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 17:05:55,190 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 17:05:55,194 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 17:05:55,279 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 17:05:55,280 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 17:13:57,874 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 17:13:57,876 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 17:13:57,957 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 17:13:57,957 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 17:28:41,293 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 17:28:41,297 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 17:28:41,379 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 17:28:41,380 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 19:22:34,011 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 19:22:34,013 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 19:22:34,098 - root - INFO - Retrieved data: [{'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'memoryType': 'Buffer', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {}, 'promptId': '8628', 'timestamp': 1738396845}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '6443', 'timestamp': 1736243872}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '4099', 'timestamp': 1736243407}, {'clientApiKey': 'sfdfsf', 'promptName': 'Simple Prompt', 'appType': 'simple', 'systemMessage': 'System message example.', 'aiMessage': 'AI message example.', 'humanMessage': 'Human message example.', 'inputData': {'key1': 'value1'}, 'promptId': '2870', 'timestamp': 1736243382}]
2025-02-03 19:22:34,099 - AiManagement.prompts - INFO - Successfully retrieved 4 prompts from the database.
2025-02-03 19:31:35,875 - absl - INFO - Using default tokenizer.
2025-02-03 19:31:35,879 - asyncio - INFO - Updating model status to Completed...
